Create ui-c8y-1020-7-0-split-cockpit-into-module-federation-plugins-6713.md (#2275)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1020-7-0-split-cockpit-into-module-federation-plugins-6713.md

---
date: ""

> title: split cockpit into module federation plugins (#6713)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Cockpit split into module federation plugins
```

> title: split cockpit into module federation plugins (#6713)

Reviewer: ```suggestion
title: Pluginification of Cockpit
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-60006
version: 1020.7.0
---

> split cockpit into module federation plugins (#6713)

Author: The Claude AI suggested to use the following body:
```suggestion
As part of the Cumulocity IoT platform, the Cockpit application provides a user interface for monitoring and controlling connected devices. To improve the modularity and maintainability of the Cockpit codebase, it has been split into separate module federation plugins. This change allows for better isolation and independent development of Cockpit features, enabling faster iteration and easier customization. Existing Cockpit functionality remains unchanged from a user perspective, ensuring a seamless transition to the new architecture.
```

> split cockpit into module federation plugins (#6713)

Reviewer: ```suggestion
The Cockpit application was splitted into further plugins. This allows admins to tailor their cockpit application to their needs without involving any coding efforts. They can remove certain plugins and therefore features which they do not want in their cockpit application. The plugins that have now been made available are: `Cockpit widgets`, `Cockpit alarms`, `Sensor phone`, `Child devices`, `Assets navigator`, `Data point library`, `Bookmarks`, `Location`, `Add location` & `Search`.
```

> The Cockpit application was splitted into further plugins. This allows admins to tailor their cockpit application to their needs without involving any coding efforts. They can remove certain plugins and therefore features which they do not want in their cockpit application. The plugins that have now been made available are: `Cockpit widgets`, `Cockpit alarms`, `Sensor phone`, `Child devices`, `Assets navigator`, `Data point library`, `Bookmarks`, `Location`, `Add location` & `Search`.

Reviewer 2: ```suggestion
The Cockpit application has been split into further plugins. This allows admins to tailor the Cockpit application to their needs without any coding effort. They can now easily remove certain plugins and thus certain features which they do not want to include in their Cockpit application. The following plugins are now available: 

* Cockpit widgets
* Cockpit alarms
* Sensor phone
* Child devices
* Assets navigator
* Data point library
* Bookmarks
* Location
* Add location
* Search
```

---
date: ""

> title: Pluginification of Cockpit

Reviewer 2: ```suggestion
title: Cockpit application split into plugins
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-60006
version: 1020.7.0
---
The Cockpit application has been split into further plugins. This allows admins to tailor the Cockpit application to their needs without any coding effort. They can now easily remove certain plugins and thus certain features which they do not want to include in their Cockpit application. The following plugins are now available: 

* Cockpit widgets
* Cockpit alarms
* Sensor phone
* Child devices
* Assets navigator
* Data point library
* Bookmarks
* Location
* Add location

> * Search

Reviewer 2: I will merge now and add your suggestion directly in the source file.
